apps/volunteers/urls.py

```diff
@@ -15,4 +15,5 @@
     path('profile/update/', views.update_profile, name='update_profile'),  # Not needed, update in profile/ only
     path('list/', views.volunteers_list, name='volunteers_list'),
     path('ajax/list/', views.ajax_volunteers_list, name='ajax_volunteers_list'),
+    path('all_volunteers/', views.VolunteerListView.as_view(), name='all_volunteers'),
```

Comment:
Let it be displayed at `/` only. Also, change `name` to "volunteers".

Reply:
okay